src/main/java/com/mraof/minestuck/block/CustomLampBlock.java

```diff
@@ -0,0 +1,37 @@
+package com.mraof.minestuck.block;
+
+import net.minecraft.core.BlockPos;
+import net.minecraft.world.InteractionHand;
+import net.minecraft.world.InteractionResult;
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.level.Level;
+import net.minecraft.world.level.block.Block;
+import net.minecraft.world.level.block.state.BlockState;
+import net.minecraft.world.level.block.state.StateDefinition;
+import net.minecraft.world.level.block.state.properties.BooleanProperty;
+import net.minecraft.world.phys.BlockHitResult;
+
+public class CustomLampBlock extends Block
+{
+	public static final BooleanProperty CLICKED = BooleanProperty.create("clicked");
```

Comment:
To maintain consistency with our other custom blockstate properties, move the creation of CLICKED to MSProperties!

Reply:
Done